core/src/main/kotlin/org/evomaster/core/search/StructuralElement.kt

```diff
@@ -192,6 +201,7 @@ abstract class StructuralElement (
 
     open fun addChildren(position: Int, list : List<StructuralElement>){
         verifyChildrenToInsert(list)
+        preChildrenSetup(list)
```

Comment:
shouldn't this be done after checking validity of inputs?

Reply:
This was changed by @ZhangMan1126